packages/pn-commons/src/components/__test__/CustomDatePicker.test.tsx

```diff
@@ -19,11 +19,20 @@ const RenderDatePicker = ({ language = 'it' }: { language?: string }) => (
 
 describe('test CustomDatePicker component', () => {
   it('renders the component', () => {
-    const { getByPlaceholderText, container } = render(<RenderDatePicker />);
-    const input = getByPlaceholderText(/datepickerinput/i);
+    const { getByLabelText, container } = render(<RenderDatePicker />);
```

Comment:
why this change?

Reply:
new component not modify placeholder and i use label

Reply:
i don't understand. texfield in RenderDatePicker has placeholder set to placeholder. why are you saying that there is no placeholder?

Reply:
it fails to take the placeholder that I insert in all tests